R/aggregate_by_matrix.R

```diff
@@ -0,0 +1,37 @@
+#' @rdname aggregate
+#'
+#' @export
+colMeansMat <- function(x, MAT)
+  colSumsMat(x, MAT)/colSums(MAT)
+
+#' @rdname aggregate
+#'
+#' @export
+colSumsMat <- function(x, MAT)
+  t(MAT) %*% x
+
+
+##' @param MAT An adjacency matrix that defines what features with
```

Comment:
At least for me it is a little bit difficult to understand: "defines what features"? defines what elements are features?

Reply:
Indeed, part of that sentence got lost. Will amend.